<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdministratorPasswordInputForUKSecurityDialog</class>
 <widget class="QDialog" name="AdministratorPasswordInputForUKSecurityDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ES2U_MAIN_069</string>
  </property>
  <widget class="QPushButton" name="OK_pushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ES2U_MAIN_046</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_pushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ES2U_MAIN_047</string>
   </property>
  </widget>
  <widget class="QLabel" name="Password_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ACCESSCONTROL_006</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Password_colon_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>:</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password_Input_lineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>147</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Explanation_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>351</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>ES2U_MAIN_070</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="UKSecurity_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>341</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>ES2U_MAIN_071</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>OK_pushButton</sender>
   <signal>pressed()</signal>
   <receiver>AdministratorPasswordInputForUKSecurityDialog</receiver>
   <slot>PushButtonAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Cancel_pushButton</sender>
   <signal>pressed()</signal>
   <receiver>AdministratorPasswordInputForUKSecurityDialog</receiver>
   <slot>PushButtonAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Password_Input_lineEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>AdministratorPasswordInputForUKSecurityDialog</receiver>
   <slot>LineEditFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>PushButtonAction()</slot>
  <slot>LineEditFinished()</slot>
 </slots>
</ui>
